### DISCUSSION TITLE
Auto memories claimer?

### DISCUSSION THREAD

**Discussion by ImpalaPUA** [RakambdaOrg/ChannelPointsMiner#377] [Q&A] [answered]
Does your version differ from the TKD-alex version and can claim memories?

**Comment by Rakambda** [comment, marked as answer]
The core principle is the same as TKD's. However yes it has its differences, mostly on the configuration.
Plus the login method is quite different with twitch restrictions that happened since then. You'll need a selenium instance to login.

Concerning moments, yes it's supported.
